mapper-runtime/src/main/java/com/datastax/oss/driver/internal/mapper/DaoBase.java

```diff
@@ -270,4 +274,19 @@ protected static void throwIfProtocolVersionV3(MapperContext context) {
               NullSavingStrategy.class.getSimpleName(), NullSavingStrategy.DO_NOT_SET.name()));
     }
   }
+
+  protected static <EntityT> void throwIfKeyspaceMissing(
```

Comment:
The jira ticket called for an error message like:

```
Missing keyspace for userDao(). Suggestions: use SessionBuilder.withKeyspace() when creating your session, specify a default keyspace on User with @Entity(defaultKeyspace), or use a DAO factory method with an @DaoKeyspace parameter
```

I was almost able to match this 1:1 except for providing the dao method name.  Fortunately that shows up in the stack trace.  I think we could probably add some context to resolve it, but wasn't sure how important that was.

Reply:
👍 if it appears in the stack trace that should be clear enough.